zulip_botserver/setup.py

```diff
@@ -51,14 +52,15 @@
 
     # Manual dependency check
     def check_dependency_manually(module_name, version=None):
+        # type: (str, Optional[str]) -> None
         try:
-            module = import_module(module_name)
+            module = import_module(module_name)  # type: Any
```

Comment:
`types.ModuleType` ?

Reply:
Same error as in `zulip_bots/setup.py`: `Module has no attribute "__version__"`.